<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnShowCars">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All Cars</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textCars">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewCar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteCar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateCar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Car</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textId">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBranch">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Branch</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textModel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Model</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
